recipes/libraw/all/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+cmake_minimum_required(VERSION 3.1)
```

Comment:
I guess the project doesn't provide any build files and we have to add it, Anyway, please add a note about it at the top of this file